react/file_watcher.py

```diff
@@ -27,10 +32,18 @@ def __init__(self, base_path, watch_patterns=None, ignore_patterns=None):
         self.last_modified_file = None
         self.last_modified_file_inc = 0
 
+    def display_content(self, dir_path):
```

Comment:
Do not need to pass `dir_path` since you can use `self.base_path`

Reply:
I would keep it as is, since:
```
self.display_content(self.base_path)
```
Reads better than:
```
self.display_content() 
# or
self.display_files_in_base()
```